File: solutions/dart/05-iz3/code/bin/main.dart
import 'dart:io';

void main(List<String> arguments) {
  while (true) {
    stdout.write('\$ ');

    // Wait for user input
    String command = stdin.readLineSync()!;
    if (command == 'exit') {
      break;
    } else if (command.startsWith('echo ')) {
      print(command.substring(5));
    } else {
      print('$command: command not found');
    }
  }
}

File: solutions/python/05-iz3/code/app/main.py
import sys


def main():
    while True:
        sys.stdout.write("$ ")

        # Wait for user input
        command = input()
        if command == "exit":
            break
        elif command.startswith("echo "):
            print(command[5:])
        else:
            print(f"{command}: command not found")


if __name__ == "__main__":
    main()
